pyprof/nvtx/nvmarker.py

```diff
@@ -71,37 +71,35 @@ def isfunc(mod, f):
 # Returns a dict string with a tracemarker and function stack in it
 #
 def traceMarker():
+
+    # Return True if the name in the hierarchy should be skipped
+    def should_skip_frame_name(name):
+        if name in ["__call__","wrapper_func","<module>", "always_benchmark_wrapper"]:
```

Comment:
I thought there were other new functions we would need to omit, but it looks like the other cases still use "wrapper_func", so we are covered. It would be nice if this was more consistent. We should either always use "wrapper_func", or prepend our functions with "pyprof" so it is easier to filter them.

No changes requested here, but maybe should create a follow-on story?